api/swagger-spec/oapi-v1.json

```diff
@@ -16987,6 +16987,10 @@
        "$ref": "v1.DeploymentCause"
       },
       "description": "extended data associated with all the causes for creating a new deployment"
+     },
```

Comment:
How many controllers update status?

On Nov 10, 2015, at 8:18 AM, Dan Mace <notifications@github.com> wrote:

In api/swagger-spec/oapi-v1.json
<https://github.com/openshift/origin/pull/5530#discussion_r44426699>:

> @@ -16987,6 +16987,10 @@
>         "$ref": "v1.DeploymentCause"
>        },
>        "description": "extended data associated with all the causes for creating a new deployment"
> +     },

The field it refers to is actually status.details.message, so it seems
misleading in two ways: it's not the time of the status field itself was
updated, and status.details.causes is updated independently elsewhere. Kind
of a mess IMO. Thoughts?

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/5530/files#r44426699>.